src/main/resources/application-fin12.yml

```diff
@@ -40,6 +40,18 @@ ams:
       host: http://localhost:8443
       repayment-path: /fineract-provider/api/v1/interoperation/transactions/{accountNumber}/loanrepayment
 
-
+mock-service:
+  local:
+    loan:
+      host: http://localhost:7070
```

Comment:
why is the port 7070? are we exposing it out?

Reply:
7070 port for mock server.